.github/PULL_REQUEST_TEMPLATE.md

```
@@ -2,20 +2,25 @@
+<!--관련 있는 Issue를 태그해주세요. (e.g. > - #1) -->
 
-_해당 PR에 대한 작업 내용을 요약하여 작성해주세요._
+<!--해당 PR에 대한 작업 내용을 요약하여 작성해주세요-->
 
 
 ## 🖇️ Tasks
 
--
-
-_해당 PR에 수행한 작업을 작성해주세요._
+- [ ] 
```

markdownlint-cli2 (0.21.0)

[warning] Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)